tensorflow/core/util/ctc/ctc_beam_search.h

```diff
@@ -239,43 +242,43 @@ float CTCBeamSearchDecoder<CTCBeamState, CTCBeamComparer>::GetTopK(
     }
   }
   // Return max value which is in 0th index or blank character logit
-  return std::max((*top_k_logits)[0], input(num_classes_ - 1));
+  return std::max((*top_k_logits)[0], input(this->num_classes_ - 1));
```

Comment:
why?

Reply:
same as in https://github.com/tensorflow/tensorflow/pull/31164#discussion_r312368512